README.md

```diff
@@ -21,3 +21,35 @@ pattern `fhir-zulip-nlp*.csv`.
 
 Releases from previous runs have been uploaded on [GitHub](https://github.com/jhu-bids/fhir-zulip-nlp-analysis/releases)
 and [GoogleDrive](https://drive.google.com/drive/u/0/folders/16MFLnKoKA5gk4ELbSnVS2GCjgR_R0ETL).
+
+### Codebook
+#### `zulip_raw_results.csv`
+TODO
+
+#### `zulip_raw_results_user_participation.csv`
+TODO
+
+#### `zulip_report_queries_with_no_results.csv`
+TODO
+
+#### `zulip_user_info.csv`
+TODO
+
+#### `zulip_report1_counts.csv`
+TODO
+
+#### `zulip_report2_thread_lengths.csv`
+TODO
+
+#### `zulip_report3_users.csv`
```

Comment:
Added a "codebook" section to the README.md. I added a codebook for the most recent report, but I need to do so for the rest of the outputs. See: #31